index.html

```diff
@@ -8,29 +8,157 @@
+    </ul>
+    <div class="scroll">
+      <img src="images/Group 55.png" class="arrow" alt="Arrow">
+      <img src="images/Work.png" class="work" alt="work">
+    </div>
+    <div class="text">MY RECENT WORK</div>
+    <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2">Multi-Post Stories</div>
+      <div class="item3">
+        <div class="bgr"><img src="images/download__1_-removebg-preview.png" alt="Ellipse"></div>  
+        A daily selection of privately<br> personalized reads; no accounts<br>
+         or sign-ups required. has been the<br> industry's standard dummy text<br>
+          ever since the 1500s, when an<br> unknown printer took a standard<br> dummy text.
+      </div>
+      <div>
+        <div class="htmlcss">
+          <div class="item4">css</div>
+          <div class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item6">html</div>
+          <div class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item8">bootstrap</div>
+          <div class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item10">ruby</div>
         </div>
-        <h1 class="name">Abednego Kibitok</h1>
-        <h2 class="title">Software Developer</h2>
-        <p class="pa">
-            I can help you build a product ,<br>
-             feature or website Look through<br>
-              some of my work and experience! If<br>
-               you like what you see and have a<br>
-                project you need coded,  don’t<br>
-                 hestiate to contact me.
-        </p>
-        <img src="images/Vector 4.png" class="line" alt="Horizontal line">
-       </section>
-   </section>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2">Multi-Post Stories<div class="bgr1"><img src="images/download__1_-removebg-preview.png" alt="Ellipse"></div></div>
+      <div class="item3">   
+        A daily selection of privately<br> personalized reads; no accounts<br>
+         or sign-ups required. has been the<br> industry's standard dummy text<br>
+          ever since the 1500s, when an<br> unknown printer took a standard<br> dummy text.
+      </div>
+      <div>
+        <div class="htmlcss">
+          <div class="item4">css</div>
+          <div class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item6">html</div>
+          <div class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item8">bootstrap</div>
+          <div class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item10">ruby</div>
+        </div>
```

Comment:
- [ ] Kindly make sure the tech stack container is wrapped into the `ul` tag and each of the is wrapped into the `li` tag in order to structure your HTML semantically correctly.